<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="folder">
         <property name="text">
          <string>Folder:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="status">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replace_image">
         <property name="text">
          <string>Replace image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replace_audio">
         <property name="text">
          <string>Replace audio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restore_audio">
         <property name="text">
          <string>Restore audio</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_manual"/>
    <addaction name="action_osz"/>
    <addaction name="action_directory"/>
    <addaction name="action_refresh"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_directory">
   <property name="text">
    <string>Set osu! directory</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>Refresh database cache</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="text">
    <string>Manually select .osu file...</string>
   </property>
  </action>
  <action name="action_osz">
   <property name="text">
    <string>Manually select .osz file...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
